ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,68 @@
+def bool_to_int(value): return int(value)
+
+
+print(bool_to_int(True))
+print(bool_to_int(False))
+
+
+def get_smaller(a, b): return min(a, b)
+
+
+print(get_smaller(16, 31))
+print(get_smaller(253, 223))
```

Comment:
The first two should be defined as lambda functions:

---

![image](https://user-images.githubusercontent.com/16143771/146486921-48a9bfe5-db0c-4625-a359-9feccdfb0061.png)